### PR TITLE
Ref/cleanup eb

### PR DESCRIPTION
**Pull request by EricBelongea**
This is the PR for when we all refactored and smoothed out some differences in our code. We made sure that all files were named the same, stat_tacker was intact, and spec files were working and expected tests were passing.

### CODE REVIEW

**Review by zkevkev — APPROVED** [review]

File: lib/game.rb
Comment (billyburch) [inline lib/game.rb:1]: Thanks for cleaning this up - should avoid further merge conflicts at this point.

File: lib/season.rb
Comment (billyburch) [inline lib/season.rb:1]: Good catch on deleting - I think decision about rewording our instance variables can happen when we refactor. Let's continue focusing on getting our code working with initialization naming as is.

File: lib/season.rb [hunk 1]
```diff
@@ -4,13 +4,11 @@ def initialize(game_team_data, game_data, team_data)
...
     @game_data = game_data
     @team_data = team_data
-    @games_data = games_data
   end
 
-
   def most_tackles(season)
     game_ids = []
-    @games_data.each do |row|
+    @game_data.each do |row|
```
Comment (vsanantone) [inline lib/season.rb:11]: I will respect this team decision on naming conventions. Glad we talked about it.

File: spec/season_spec.rb
Comment (billyburch) [inline spec/season_spec.rb:1]: Good catch on keeping instance names consistent until a group refactor can happen.

File: spec/season_spec.rb [hunk 2]
```diff
@@ -53,12 +53,12 @@
     expect(@season.least_accurate_team("20142015")).to eq "Columbus Crew SC"
   end
 
-  xit "#most_tackles" do
+  it "#most_tackles" do
```
Comment (vsanantone) [inline spec/season_spec.rb:56]: This is a great opportunity where we can add test for every season we have.

**Review by billyburch — APPROVED** [review]

File: spec/season_spec.rb [hunk 3]
```diff
@@ -53,12 +53,12 @@
...
     expect(@season.most_tackles("20132014")).to eq "FC Cincinnati"
     expect(@season.most_tackles("20142015")).to eq "Seattle Sounders FC"
   end
 
-  xit "#fewest_tackles" do
+  it "#fewest_tackles" do
```
Comment (vsanantone) [inline spec/season_spec.rb:61]: The same time of expansion for more robusting testing can be added here easily.

**Pull request merged**